README.md

```diff
@@ -4,16 +4,67 @@
 
 ## Usage
 
+Formulator is a library to give you:
+* A label for your input, based on the attribute name
+* An error label
+* A class around the input if there is an error for the attribute.
```

Comment:
Maybe explain why this is useful? "This allows you to easily style inputs when there is a form error"? Maybe that's obvious though :S